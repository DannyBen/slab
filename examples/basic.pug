div Some text.

div
  .a Some text.
