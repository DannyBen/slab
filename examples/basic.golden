<div>
    Some text.
</div>
<div>
    <div class="a">
        Some text.
    </div>
</div>
